### PR TITLE
[BPK-4165] Allow banner-alert to accept node or string for message

### PR DESCRIPTION
**Pull request by georgegillams**
+ [x] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-react-native/blob/master/CONTRIBUTING.md)

Remember to include the following changes:
+ [x] `UNRELEASED.md`
+ [x] `README.md`
+ [x] Tests
+ [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/master/CODE_REVIEW_GUIDELINES.md)_

Turns out the iOS font “Didot” has the same emoji problem in RN

image

### CODE REVIEW

**Review by shaundon — CHANGES_REQUESTED** [review]
The code changes look good but I think the documentation + testing side could be a bit more comprehensive:

* Can we have a snapshot test for using a Node as the message prop?
* The story example is quite specific to the emoji case, which is going to end up being covered by `BpkEmoji` when that's created. How about making the story more generic, something like a `BpkText` with a bit of bold text embedded within? Or having that in addition to the current story.

**Review by shaundon — CHANGES_REQUESTED** [review]
Prop types table in readme needs updating too

File: lib/bpk-component-banner-alert/stories.js [hunk 1]
```diff
@@ -399,6 +399,18 @@ storiesOf('bpk-component-banner-alert', module)
...
     return (
       <View>
+        <BpkBannerAlert
+          style={styles.bannerAlert}
+          type={ALERT_TYPES.neutral}
+          message={
+            <>
+              Your booking was successful. Your booking reference is{' '}
+              <BpkText textStyle="sm" weight="emphasized">
+                fr33sh4v0c4d0
```
Comment (shaundon) [inline lib/bpk-component-banner-alert/stories.js:409]: omg you used my example absolute madlad

**Review by shaundon — CHANGES_REQUESTED** [review]
still needs a Jest snapshot but otherwise good

File: lib/bpk-component-banner-alert/stories.js (hunk 1 above)
Comment (georgegillams) [reply lib/bpk-component-banner-alert/stories.js:409]: I loved it

Comment (georgegillams) [reply lib/bpk-component-banner-alert/stories.js:409]: Simulator Screen Shot - iPhone 11 Pro Max - 2020-08-27 at 09 34 54

Comment (shaundon) [reply lib/bpk-component-banner-alert/stories.js:409]: get down to del taco

Comment (samstarling) [reply lib/bpk-component-banner-alert/stories.js:409]: You love to see it

**Review by shaundon — APPROVED** [review]

**Pull request merged**